### ISSUE TITLE
JSS interface

### ISSUE THREAD

**Issue by samsch** [airbnb/react-with-styles#150]
The linked JSS interface in the README is not a usable npm package, released, or maintained.

https://github.com/oliviertassinari/react-with-styles-interface-jss/issues/3

Perhaps a note should be added to readme to state as such? It's a bit misleading, and sent me on a bit of a chase to make react-with-styles (for react-dates) work with MUI's JSS.

**Comment by ljharb** [comment]
That's unfortunate; based on https://github.com/oliviertassinari/react-with-styles-interface-jss/issues/3#issuecomment-336170632 we shouldn't be listing it at all.

@oliviertassinari, would you be willing to hand over the npm name if you're not interested in maintaining a package with it? That way, if others want to fork and improve yours, or write a replacement, they're able to do so.

**Comment by oliviertassinari** [comment]
Yes sure. I have been working on a custom wrapper   in top of JSS since then (`@material-ui/core/styles`) that can be used standalone.

**Comment by ljharb** [comment]
Thanks, if you want to `npm owner add airbnb` then we can coordinate with whoever wants to work on it.

**Comment by oliviertassinari** [comment]
`npm info react-with-styles-interface-jss`
> maintainers:
> - airbnb <jordan.harband+npm@airbnb.com>
> - oliviertassinari <olivier.tassinari@gmail.com>
